.github/workflows/cron_nightly_tests.yml

```diff
@@ -0,0 +1,185 @@
+        run: npm run ${{ env.TEST_CAMPAIGN }}
+
+      # Rename and upload report as artifact
+      - name: Rename file reports
+        working-directory: ${{ env.TESTS_DIR }}
+        env:
+          REPORTS_DIR_PATH: ${{ github.workspace }}/${{ env.REPORTS_DIR }}
+        run: |
+          mkdir -p ${{ env.REPORTS_DIR_PATH }}
+          FILENAME="$( echo -e '${{ env.TEST_CAMPAIGN }}' | tr  ':' '-'  )"
+          mv ./mochawesome-report/mochawesome.json ${{ env.REPORTS_DIR_PATH }}/${FILENAME}.json
+
+      - uses: actions/upload-artifact@v2
+        name: Upload report
+        with:
+          name: reports_${{ env.GH_BRANCH }}
+          path: ${{ env.REPORTS_DIR }}
+
+  # Second job: Merge Mochawesome reports and upload them on GCP
+  merge-upload-reports:
+    name: Combine all reports and upload them
+    if: ${{ always() }}
+    needs:
+      - test
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        BRANCH:
+          - develop
+
+    env:
+      GH_BRANCH: ${{ matrix.branch }}  # Branch to test
+      REPORTS_DIR: 'reports_${{ matrix.BRANCH }}'  # Where to move test's reports
+      COMBINED_REPORT_DIR: 'combined_reports'  # Where to store the combined report
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ env.GH_BRANCH }}
+
+      - uses: actions/download-artifact@v2
+        name: Download reports
+        with:
+          name: reports_${{ env.GH_BRANCH }}
+          path: ${{ env.REPORTS_DIR }}
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install dependencies
+        working-directory: ${{ env.TESTS_DIR }}
+        run: npm install
+
+      - name: Combine reports
+        working-directory: ${{ env.TESTS_DIR }}
+        env:
+          REPORTS_DIR_PATH: ${{ github.workspace }}/${{ env.REPORTS_DIR }}
+          COMBINED_REPORT_DIR_PATH: ${{ github.workspace }}/${{ env.COMBINED_REPORT_DIR }}
+        run: |
+          mkdir -p ${{ env.COMBINED_REPORT_DIR_PATH }}
+          REPORT_NAME="$(date +%Y-%m-%d)-${{ env.GH_BRANCH }}"
+          npx mochawesome-merge "${{ env.REPORTS_DIR_PATH }}/*.json" -o "${{ env.COMBINED_REPORT_DIR_PATH }}/${REPORT_NAME}.json"
+
+      - uses: google-github-actions/setup-gcloud@master
+        name: Setup Cloud Sdk
+        with:
+          service_account_key: ${{ env.GC_SERVICE_KEY }}
+          project_id: ${{ env.GC_PROJECT_ID }}
+
+      - name: Upload to Google Cloud Storage (GCS)
+        run: gsutil cp -r "${{ env.COMBINED_REPORT_DIR }}/**" gs://prestashop-core-nightly/reports
+
+  # Add pushed reports to GCp on nightly
+  push-nightly-reports:
+    name: Push reports to nightly.prestashop.com
+    if: ${{ always() }}
+    needs:
+      - merge-upload-reports
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        BRANCH:
+          - develop
```

Comment:
As said on the Pull request description, 1.7.8.x will continue to run on GCP because there are some updates to do before migrating to Github action (the docker file, UI tests ...)